src/Quartz/JobBuilder.cs

```diff
@@ -157,7 +159,7 @@ public IJobDetail Build()
             }
 
             return new JobDetailImpl(Key ?? new JobKey(Guid.NewGuid().ToString()),
-                                     jobType,
+                                     new JobType(jobTypeSerialized),
```

Comment:
implicit conversion operator between Type and and JobType should make this redundant